src/components/step/step.css

```diff
@@ -0,0 +1,54 @@
+html {
```

Comment:
(I would have this file be attached to my proposed ```<StepLayout>```) component so it's StepLayout.css)